tests/scaletest/lib/utils.go

```diff
@@ -213,3 +213,45 @@ func FetchPools(t *testing.T, AviClient *clients.AviClient) []models.Pool {
 	}
 	return pools
 }
+
+func FetchDnsVsFqdn(t *testing.T, dnsVsUuid string, AviClient *clients.AviClient) []models.DNSRecord {
+	uri := "api/virtualservice"
+	result, err := AviClient.AviSession.GetCollectionRaw(uri)
+	if err != nil {
+		t.Logf("Get uri %v returned err for pool %v", uri, err)
```

Comment:
Please change this to Fatalf or return after this line.

Reply:
Done